src/components/LowerLayout/GeneralInfo/index.js

```diff
@@ -10,23 +9,26 @@ const createList = listContainer => listContainer.map(({ list }) => (
 ));
 
 const createLink = additionalInfo => additionalInfo.map(({ link }) => {
-  if (!link) {
-    return null;
+  if (link) {
```

Comment:
It's good practice to always have a function return something (even if null). In addition, it is also a good practice to list all the 'fail' conditions upfront. So I would do the following:

```
if (!link) {
  return null;
}

return .......
```

Reply:
I see. I changed it to a shorter version not knowing any better. Thanks for the tip :) Especially for the fail ones. I didn't know about that at all.